_layouts/progress-page.html

```diff
@@ -55,7 +55,7 @@ <h3 class="milestone-quarter-heading" id="milestone-q1">Quarter 1, 2020 (Jan - M
 		</div>
 	</div>
 
-	<div class="usa-grid milestone-progress-row milestone-current status-complete" id="milestone-7-1">
+	<div id="action-7-1" class="usa-grid milestone-progress-row milestone-current status-complete" id="milestone-7-1">
```

Comment:
This line is failing because `id` is defined twice.
```suggestion
	<div id="action-7-1" class="usa-grid milestone-progress-row milestone-current status-complete">
```